.github/workflows/run-on-arm.yml

```diff
@@ -74,13 +74,13 @@ jobs:
             python3 -m pip list
 
             set -e
-            git clone https://github.com/Snaipe/Criterion.git -b v2.3.3
-            cd Criterion
-            git submodule update --init
+            cd /tmp && wget https://github.com/Snaipe/Criterion/releases/download/v2.3.3/criterion-v2.3.3.tar.bz2
```

Comment:
I see the benefit of it: logic is not copied, if new Criterion is released it would available instantly, though it would require some changes to dbld: the scripts are expected to run inside the container: e.g. this is in dbld/builddeps: 
```
. /dbld/functions.sh
```